EIPS/eip-2936.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 2936
+title: EXTCLEAR Opcode For SELFDESTRUCTed contracts
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2936-extclear-for-selfdestruct/4569
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-03
+replaces: 2751
+---
+
+`EXTCLEAR` Opcode For `SELFDESTRUCT`ed contracts
+
+## Simple Summary
+Enable new opcode to clear storage for `SELFDESTRUCTED`ed contracts.
+
+## Abstract
+`SELFDESTRUCT` complexity can be reduced by deferring state cleanup.
+
+## Motivation
+`SELFDESTRUCT` (`0xFF`) is unnecessarily complex because it clears an unbounded amount of contract storage.
+It is computationally expensive for nodes to track all of the storage used in every contract in case the contract `SELFDESTRUCT`s.
+Further, contracts can be re-initialized using `CREATE2` (`0xF5`), and then `SLOAD` (`0x54`) prior storage.
+Therefore, several ethereum clients do not clear storage at all, and just check if the contract was initiated since `SSTORE` (`0x55`) during `SLOAD`.
+Nobody expected `SELFDESTRUCT` and `CREATE2` to increase the cost of `SLOAD`.
```

Comment:
Would be nice to get clarification on what this means and how it relates to the motivation of this EIP.

Reply:
clarified in 5d079a18a2c77d8685812e1759dce1393e46e6d4